index.ts

```diff
@@ -469,12 +467,6 @@ function getProjectFiles(rootDir: string, callback: FilesCallback): void {
         if (err) return callback(new Error(err), null, null);
         // Check if there are any .gs files
         // We will prompt the user to rename files
-        let canRenameToJS = false;
```

Comment:
Can we comment this block out and add a TODO here?

This feature is incomplete. It was intended to rename files from `gs` to `js`.